plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrinterStatus.py

```diff
@@ -65,13 +65,7 @@ def updateOutputModel(self, model: PrinterOutputModel) -> None:
         model.updateName(self.friendly_name)
         model.updateType(self.machine_variant)
         model.updateState(self.status if self.enabled else "disabled")
-
-        # Make sure to set the build plate even though we don't use it. Since it's optional, use
-        # glass as a default
-        if self.build_plate:
-            model.updateBuildplate(self.build_plate.type)
-        else:
-            model.updateBuildplate("glass")
+        model.updateBuildplate(self.build_plate.type or "glass")
```

Comment:
mm like this it will fail if self.build_plate is None. Should be `self.build_plate.type if self.build_plate else "glass"`.